src/util.js

```diff
@@ -14,12 +14,17 @@ const focusableElementsList = [
     '*[contenteditable]'
 ];
 
-function createTrapBoundary() {
-    const div = document.createElement('div');
-    div.setAttribute('tabindex', '0');
-    div.className = 'keyboard-trap-boundary';
+let trapBoundary;
+if (typeof document !== "undefined") {
+    trapBoundary = document.createElement('div');
+    trapBoundary.setAttribute('tabindex', '0');
+    trapBoundary.className = 'keyboard-trap-boundary';
+}
 
-    return div;
+const noop = { addEventListener: () => {} };
+
+function createTrapBoundary() {
+    return trapBoundary ? trapBoundary.cloneNode() : noop;
```

Comment:
Instead of calling noop, maybe we could call a private _createTrapBoundary() method. This means the initial trap boundary element is instantiated lazily.

Reply:
I see what you mean, and it can be created lazily, but the `noop` case is for when this is run on the server, so we cannot create any dom elements. Perhaps a comment is in order :)

Reply:
Cool. Let's create it lazily and keep the noop case if possible.